hapic/processor/main.py

```diff
@@ -103,6 +103,27 @@ def generate_schema_ref(
         eg. {"$ref": "#/definitions/MySchema"}
         """
 
+    @classmethod
+    def schema_class_resolver(
+        cls,
+        main_plugin: BasePlugin,
+        schema: 'TYPE_SCHEMA',
+    ) -> 'TYPE_SCHEMA':
+        """
+        Return schema class with adaptation if needed.
+        :param main_plugin: associated Apispec plugin
+        :param schema: schema to proceed
+        :return: schema generated from given schema or original schema if
+            no change required.
+        """
+        """
```

Comment:
Why 2 docstring block at begin of method ?

Reply:
copy paste error of course